proto/api.go

```diff
@@ -79,6 +79,7 @@ type Request interface {
 	gogoproto.Message
 	// Header returns the request header.
 	Header() *RequestHeader
+	KVHeader() *KVRequestHeader
```

Comment:
I'll go ahead and split that interface and see where it take me. Thanks for pointing me to the stdlib